CHANGELOG.md

```diff
@@ -1,5 +1,58 @@
 # Release Notes
 
+## Version 1.12.2 - 2020-08-03
+
+Features:
+* Automated GCP Credentials Addon Prototype [#8682](https://github.com/kubernetes/minikube/pull/8682)
```

Comment:
Change this to
New Addon: and remove prototype word

Reply:
done